app/src/main/java/com/eventyay/organizer/core/settings/restriction/CheckInRestrictions.java

```diff
@@ -100,7 +100,7 @@ private void checkRestrictAll() {
         boolean restrictAll = true;
 
         for (Ticket ticket : ticketSettingsViewModel.getTickets().getValue()) {
-            if (ticket.isCheckinRestricted == null || !ticket.isCheckinRestricted) {
+            if (!ticket.isCheckinRestricted) {
```

Comment:
Why? What if it's null?

Reply:
I changed the type from Boolean to boolean. A **boolean** can never be null. It can be either true or false.